traveltimepy/dto/common.py

```diff
@@ -140,3 +140,18 @@ class LevelOfDetail(BaseModel):
     scale_type: Literal["simple", "simple_numeric", "coarse_grid"] = "simple"
     level: Optional[Union[int, str]]
     square_size: Optional[int]
+
+
+class TimeInfo:
+    def __init__(self, time_value: datetime):
+        if not isinstance(time_value, datetime):
+            raise ValueError("time must be a datetime object")
+        self.time = time_value
```

Comment:
it can be value

Reply:
I don't think I can avoid using the __init__ method completely if that's what you mean. But simplified it a bit